sos/plugins/openstack_glance.py

```diff
@@ -2,6 +2,7 @@
 # Copyright (C) 2012 Rackspace US, Inc.,
```

Comment:
The commit subject is misleading - if you are going to change all the OpenStack plugins in one patch, use `[openstack_*]` as the commit tag. Otherwise you should never e.g. have hunks that apply to `openstack_glance.py` in a patch with a subject of `[openstack_instack]` - it's just confusing for anyone trying to understand the logs down the line.

Your other option here would be to make one commit per file - this is especially preferred if you are making other sets of unrelated changes (e.g. the very large addition to `openstack_insight.py` vs. the relatively minore env. var. change in all the others).

I'm not going to make you respin and split up the patch series at this stage, but it is the preferred style in `sos` (and many other projects), but the commit subject definitely needs fixing before this can go in.